tinker/faculty_bios/forms.py

```diff
@@ -55,7 +55,8 @@ def validate_username(form, field):
     username = field.data
     host = "http://wsapi.bethel.edu"
     path = "/username/" + username + "/roles"
-    req = requests.get(host + path)
+    req = FacultyBioController.tinker_requests(host + path)
```

Comment:
can you actually clean up this code? Make the host and path be set in one line.
`url = "http://wsapi.bethel.edu/username/%s/roles" % username

Reply:
Done